py/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleated
+
+- add rudimentary mypy setup. ([#2384](https://github.com/getsentry/relay/pull/2384))
```

Comment:
```suggestion
## Unreleased

- Add rudimentary Mypy setup. ([#2384](https://github.com/getsentry/relay/pull/2384))
```

Reply:
Based on https://github.com/getsentry/relay#instructions-for-changelogs, I'd skip this PR from the changelog.

Reply:
this does have slight external effects in that mypy will notice symbols from this package now